elrond-wasm-debug/src/abi_json/endpoint_abi_json.rs

```diff
@@ -0,0 +1,78 @@
+use alloc::vec::Vec;
+use elrond_wasm::abi::*;
+
+use super::*;
+use serde::de::{self, Deserializer, MapAccess, Visitor};
+use serde::ser::{SerializeMap, Serializer};
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+use std::fmt;
+
+#[derive(Serialize, Deserialize)]
+pub struct InputAbiJson {
+	#[serde(rename = "name")]
+	pub arg_name: String,
+	#[serde(rename = "type")]
+	pub type_name: String,
+	/// Bool that is only serialized when true
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub multi_arg: Option<bool>,
+}
+
+impl From<&InputAbi> for InputAbiJson {
+	fn from(abi: &InputAbi) -> Self {
+		InputAbiJson {
+			arg_name: abi.arg_name.to_string(),
+			type_name: abi.type_name.clone(),
+			multi_arg: if abi.multi_arg { Some(true) } else { None },
+		}
+	}
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct OutputAbiJson {
+	#[serde(rename = "type")]
+	pub type_name: String,
+	/// Bool that is only serialized when true
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub multi_result: Option<bool>,
+}
+
+impl From<&OutputAbi> for OutputAbiJson {
+	fn from(abi: &OutputAbi) -> Self {
+		OutputAbiJson {
+			type_name: abi.type_name.clone(),
+			multi_result: if abi.multi_result { Some(true) } else { None },
```

Comment:
Same here, why Option instead of simply bool?

Reply:
same